org.eclipse.scanning.test/src/org/eclipse/scanning/test/epics/EpicsV4ConnectorTest.java

```diff
@@ -145,10 +154,10 @@ public void connectToInvalidDevice() throws Exception {
 
 		try {
 			// Get the device
-			IMalcolmDevice<EPICSv4ExampleModel> modelledDevice = service.getDevice("INVALID_DEVICE");
+			IMalcolmDevice<?> invalidDevice = createMalcolmDevice("INVALID_DEVICE");
 
 			// Get the device state. This should fail as the device does not exist
-			modelledDevice.getDeviceState();
+			invalidDevice.getDeviceState();
 
 			fail("No exception thrown but one was expected");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "No exception thrown but one was expected" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)